src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -948,8 +948,8 @@ var TorrentsTable = new Class({
 
         // priority
         this.columns['priority'].updateTd = function(td, row) {
```

Comment:
Wait i'll revert the `queue_position` because i can't change it in https://github.com/qbittorrent/qBittorrent/blob/e1f19b7c7531b4d3bd1b0b83e84302c42c77f210/src/webui/www/private/scripts/dynamicTable.js#L784
Also found something else that i'll mention when i push in a few minutes.

Reply:
Ok rebased, reverted `queue_position` and found this little bugger that i previously missed, everything was broken because of it https://github.com/qbittorrent/qBittorrent/pull/9995/files#diff-700c2fec148a222fce14d3faa49b7a62R68

Reply:
@Piccirello, is it fixed now?

Reply:
@thalieht 
There are still a few `'priority'` instances in this file, is it possible to replace them without breaking anything?

Reply:
@Chocobo1 no.